src/js/models/layerstructure.js

```diff
@@ -609,6 +609,20 @@ define(function (require, exports, module) {
     LayerStructure.prototype.byID = function (id) {
         return this.layers.get(id, null);
     };
+    
+    /**
+     * Get a list of Layer models by layer IDs.
+     *
+     * @param {Immutable.Iterable.<number>} ids
+     * @return {Immutable.Iterable.<Layer>}
+     */
+    LayerStructure.prototype.byIDs = function (ids) {
+        return ids.reduce(function (layers, id) {
```

Comment:
It would be better to use map/filter here instead of reduce, since you're creating n Immutable.Lists every time you call this function.

```
return ids
    .map(this.byID, this)
    .filter(function (layer) {
        return layer;
    });
```

or something to that effect.